code/test/Core.Test/Naming/NamingTest.cs

```diff
@@ -205,6 +206,28 @@ public void Validate_BadFormat_StartWithNumber(string language)
             Assert.Equal(ValidationErrorType.BadFormat, result.ErrorType);
         }
 
+
+        [Fact]
+        public void DirectoryExistsValidator()
```

Comment:
`DirectoryExistsValidator` is a bad name for a test. It doesn't say what it's testing (or why)